aziotctl/src/internal/check/checks/well_formed_configs.rs

```diff
@@ -4,7 +4,8 @@ use anyhow::{Error, Result};
 use serde::Serialize;
 
 use crate::internal::check::{CheckResult, Checker, CheckerCache, CheckerMeta, CheckerShared};
-
+use nix::unistd::{Gid, Group, Uid, User};
```

Comment:
Move this to be in the group of third-party imports, ie with `anyhow` and `serde` above. Also make sure to sort it, ie put it between `anyhow` and `serde`.